cmd/occollector/app/builder/receiver_builder.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018, OpenCensus Authors
```

Comment:
Why did you move the code from the other file here? Perhaps a later move as it makes the PR quite noisy.